example_config_class.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ThingWorker
+  def self.process(message)
+    FastlyNsq.logger.info("IN PROCESS: #{message}")
+  end
+end
+
+FastlyNsq.configure do |config|
+  config.channel = 'william'
+  # config.logger = Logger.new
+  # config.preprocessor = ->(message) { FastlyNsq.logger.info 'PREPROCESSESES' }
+
+  config.listen_to do |topics|
+    topics.add('assign_supplemental_plans', ThingWorker)
+  end
```

Comment:
It seems as if this could be `listen_to topic: Worker` (as in: give a hash, where topics are keys, workers are values). Are there advantages I'm not seeing to the block mechanism here? I also tend to think it's potentially confusing, just curious what the upsides might be.